build_tools/github_actions/test_executable_scripts/test_rocwmma.py

```diff
@@ -32,6 +32,12 @@
 
 test_subdir = ""
 timeout = "3600"
+negative_filter = []
+
+# skipping failed tests here https://github.com/ROCm/TheRock/pull/2533
```

Comment:
Should there should be an issue reported on the problematic tests, and then reference that issue here?